azurerm/internal/services/network/network_connection_monitor_resource.go

```diff
@@ -200,10 +226,50 @@ func resourceNetworkConnectionMonitor() *schema.Resource {
 							},
 						},
 
+						"included_ip_addresses": {
+							Type:     schema.TypeSet,
+							Optional: true,
+							Elem: &schema.Schema{
+								Type: schema.TypeString,
+								ValidateFunc: validation.Any(
+									validation.IsIPv4Address,
+									validation.IsIPv6Address,
+									validation.IsCIDR,
+								),
+							},
+						},
+
+						"target_resource_id": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Computed: true,
+							ValidateFunc: validation.Any(
+								computeValidate.VirtualMachineID,
+								logAnalyticsValidate.LogAnalyticsWorkspaceID,
+								networkValidate.SubnetID,
+								networkValidate.VirtualNetworkID,
+							),
+						},
+
+						"type": {
```

Comment:
this might be far more clear as 
```suggestion
						"target_resource_type": {
```

Reply:
Updated